helpers/checkpoint_logger/flows.py

```diff
@@ -55,3 +55,24 @@ class UploadFlow(BaseFlow):
     NOTIF_TOO_MANY_RETRIES = auto()
     NOTIF_STALE_HEAD = auto()
     NOTIF_ERROR_NO_REPORT = auto()
+
+
+@failure_events("TEST_RESULTS_ERROR")
+@success_events("TEST_RESULTS_BEGIN")
+@subflows(
+    ("notification_latency", "TEST_RESULTS_BEGIN", "TEST_RESULTS_NOTIFY"),
```

Comment:
what's the difference between "notification_latency" and "test_results_notification_latency" subflows?

Reply:
that was a mistake :P